src/app/components/home/home.component.html

```diff
@@ -0,0 +1,34 @@
+@if (trips$ | async; as trips) {
+  <div class="flex flex-col items-center overflow-x-hidden p-4">
+    @for (trip of trips; track trip.id) {
+      <div class="boundary relative flex w-[150%] justify-center">
+        <div
+          class="h-calc absolute z-[-1] flex w-[60%] overflow-hidden rounded-xl">
+          <div class="w-1/2 bg-green-500"></div>
+          <div class="w-1/2 bg-red-500"></div>
+        </div>
+        <div
+          swipe
+          (left)="onSwipeLeft(trip.title)"
+          (right)="onSwipeRight(trip.title)"
+          class="w-[60%]"
+          cdkDrag
+          cdkDragLockAxis="x"
+          (cdkDragReleased)="resetPosition($event)"
+          cdkDragBoundary=".boundary">
+          <app-trip-card
+            [tripTitle]="trip.title"
+            [startDate]="trip.startDate"
+            [endDate]="trip.endDate"
+            [totalDuration]="trip.totalDuration"
+            [totalCost]="trip.totalCost"
+            (click)="handleTripClick(trip.title)" />
+        </div>
+      </div>
+    }
+    <button
+      class="btn btn-circle fixed bottom-4 right-4 border-2 border-blue-500">
+      +
+    </button>
+  </div>
```

Comment:
You could potentially use ```w-3/5``` as opposed to an arbitrary value to set the width of the element to 60%.

Reply:
Didn't notice that, thank you.